lib/src/data/models/movie_model.dart

```diff
@@ -0,0 +1,64 @@
+class MovieModel {
+  final String title;
+  final String originalTitle;
+  final String originalLanguage;
+  final int id;
+  final bool adult;
+  final String releaseDate;
+  final String overview;
+  final double voteAverage;
+  final List<int> genres;
+  final String posterPath;
+  final String backdropPath;
+  final double popularity;
+  final bool video;
+  final int voteCount;
+
+  MovieModel({
+    required this.title,
+    required this.originalTitle,
+    required this.originalLanguage,
+    required this.id,
+    required this.adult,
+    required this.releaseDate,
+    required this.overview,
+    required this.voteAverage,
+    required this.genres,
+    required this.posterPath,
+    required this.backdropPath,
+    required this.popularity,
+    required this.video,
+    required this.voteCount,
+  });
+
+  factory MovieModel.fromJson(Map<String, dynamic> json) {
+    return MovieModel(
```

Comment:
Why for some parameter you are defining the type?

Reply:
To be honest, I don't know 🥲 , trying to be more specific about the type, but it doesn't have sense, already changed 👌